data/secscan_model/test/test_secscan_interface.py

```diff
@@ -47,7 +47,7 @@ def test_load_security_information(repository, v4_whitelist, initialized_db):
     "next_token, expected_next_token",
     [
         (None, SplitScanToken("v4", V4ScanToken(56))),
-        (SplitScanToken("v4", V4ScanToken(1)), SplitScanToken("v4", None)),
```

Comment:
We still need to test this case.

Reply:
Is it a valid case though? Looking at what `perform_indexing` does, it looks like the it will always return a token if there's a `Manifest` with a higher `id` than the one parameterized for that test.

We can see from the previous one that there is one with `id=56`...

Reply:
And fallback to `v2` if there is not.